src/components/workspace/MCQChooser.tsx

```diff
@@ -4,15 +4,30 @@ import * as React from 'react'
 import { IMCQQuestion } from '../assessment/assessmentShape'
 
 export interface IMCQChooserProps {
+  chosenOption: number | null
   mcq: IMCQQuestion
-  mcqSubmit?: (choiceId: number) => void
+  handleMCQSubmit: (choiceId: number) => void
 }
 
-class MCQChooser extends React.PureComponent<IMCQChooserProps, {}> {
+type State = {
```

Comment:
Local state preserves the mcq option, that might seem to 'flutter' back to its original value in between saving and the state being updated.